validate.go

```diff
@@ -166,16 +168,6 @@ func (r *ValidateResult) Add(path *PathBuffer, v any, msg string) {
 	})
 }
 
-// Addf adds an error to the validation result at the given path and with
-// the given value, allowing for fmt.Printf-style formatting.
-func (r *ValidateResult) Addf(path *PathBuffer, v any, format string, args ...any) {
```

Comment:
Given this is part of the public API and we can't know if others are depending on it, I don't think this can be removed even if we stop using it internally, just FYI.

Reply:
oh. You are right! Sorry about that.